src/plugins/Invites/controllers/commands/setInvite.js

```diff
@@ -0,0 +1,29 @@
+const Discord = require("discord.js");
+const dbh = require("../../../.common/structures/DataHandling/DatabaseHandler.js");
+
+exports.help = {
+  name: "addInvite",
+  description: "Adds a keyword to an invite.",
+  usage: "addInvite [name] [code]",
+  example: "addInvite website EAZfGWvN83",
+};
+
+exports.conf = {
+  aliases: [],
+  cooldown: 5,
+  permissions: ["ADMINISTRATOR"],
+  arguments: ["Name", "Code"],
+};
+
+exports.run = async (client, message, args) => {
+  let name = args[0].toLowerCase();
+  let code = args[1];
+
+  const res = await dbh.invite.setInvite(message.guild.id, name, code);
+
+  message.channel.sendEmbed({
+    color: 0xff9f01,
```

Comment:
I think this can be a constant. I can't tell what color it is just by reading it.

Reply:
That is the bot's default embed color, which is orange. We will make it a default for all embeds in a later PR. 
Embed Builder: https://github.com/dev-launchers-sandbox/project__discord-bot/blob/implement-guild/src/plugins/.common/extends/TextChannelExtension.js#L12